domains/authorization/Proxy/Controllers/CertificatesController.cs

```diff
@@ -0,0 +1,94 @@
+    [ProducesResponseType(typeof(void), StatusCodes.Status403Forbidden)]
+    [ProducesResponseType(typeof(ResultList<GranularCertificate>), StatusCodes.Status200OK)]
+    public async Task GetCertificatesV2([FromQuery] GetCertificatesQueryParameters param, [FromQuery] string? organizationId)
+    {
+        await ProxyClientCredentialsRequest("v1/certificates", organizationId);
+    }
+
+    /// <summary>
+    /// Gets all certificates in the wallet that are <b>available</b> for use.
+    /// </summary>
+    /// <response code="200">Returns the aggregated claims.</response>
+    /// <response code="401">If the user is not authenticated.</response>
+    [HttpGet]
+    [Route("v1/certificates")]
+    [Produces("application/json")]
+    [Authorize(policy: Policy.B2CSubTypeUserPolicy)]
+    [ApiVersionNeutral]
+    [Obsolete]
+    [ProducesResponseType(typeof(void), StatusCodes.Status401Unauthorized)]
+    [ProducesResponseType(typeof(void), StatusCodes.Status403Forbidden)]
+    [ProducesResponseType(typeof(ResultList<GranularCertificate>), StatusCodes.Status200OK)]
+    public async Task GetCertificates([FromQuery] GetCertificatesQueryParameters param)
+    {
+        await ProxyTokenValidationRequest("v1/certificates");
+    }
+
+    /// <summary>
+    /// Returns aggregates certificates that are <b>available</b> to use, based on the specified time zone and time range.
+    /// </summary>
+    /// <response code="200">Returns the aggregated claims.</response>
+    /// <response code="400">If the time zone is invalid.</response>
+    /// <response code="401">If the user is not authenticated.</response>
+    [HttpGet]
+    [Route("aggregate-certificates")]
+    [Produces("application/json")]
+    [Authorize(policy: Policy.B2CPolicy)]
+    [ApiVersion(ApiVersions.Version20250101, Deprecated = true)]
+    [ProducesResponseType(typeof(void), StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(typeof(void), StatusCodes.Status401Unauthorized)]
+    [ProducesResponseType(typeof(void), StatusCodes.Status403Forbidden)]
+    [ProducesResponseType(typeof(ResultList<AggregatedCertificates>), StatusCodes.Status200OK)]
+    public async Task AggregateCertificatesV2([FromQuery] AggregateCertificatesQueryParameters param, [FromQuery] string? organizationId)
+    {
+        await ProxyClientCredentialsRequest("v1/aggregate-certificates", organizationId);
+    }
+
+    /// <summary>
+    /// Returns aggregates certificates that are <b>available</b> to use, based on the specified time zone and time range.
+    /// </summary>
+    /// <response code="200">Returns the aggregated claims.</response>
+    /// <response code="400">If the time zone is invalid.</response>
+    /// <response code="401">If the user is not authenticated.</response>
+    [HttpGet]
+    [Route("v1/aggregate-certificates")]
+    [Produces("application/json")]
+    [Authorize(policy: Policy.B2CSubTypeUserPolicy)]
+    [ApiVersionNeutral]
+    [Obsolete]
```

Comment:
Why is this obsolete? 🤔

Reply:
Its V1/ endpoint

Reply:
Then deprecated is the annotation we should use no?